imageviewer.js

```diff
@@ -29,570 +31,573 @@
-            window.requestAnimationFrame = window[vendors[x] + 'RequestAnimationFrame'];
-            window.cancelAnimationFrame = window[vendors[x] + 'CancelAnimationFrame'] || window[vendors[x] + 'CancelRequestAnimationFrame'];
+            window.requestAnimationFrame = window[vendors[x] + 'RequestAnimationFrame']
+            window.cancelAnimationFrame =
+                window[vendors[x] + 'CancelAnimationFrame'] || window[vendors[x] + 'CancelRequestAnimationFrame']
         }
 
         if (!window.requestAnimationFrame)
-            window.requestAnimationFrame = function (callback, element) {
-                var currTime = new Date().getTime();
-                var timeToCall = Math.max(0, 16 - (currTime - lastTime));
-                var id = window.setTimeout(function () {
-                        callback(currTime + timeToCall);
-                    },
-                    timeToCall);
-                lastTime = currTime + timeToCall;
-                return id;
-            };
+            window.requestAnimationFrame = function(callback, element) {
+                var currTime = new Date().getTime()
+                var timeToCall = Math.max(0, 16 - (currTime - lastTime))
+                var id = window.setTimeout(function() {
+                    callback(currTime + timeToCall)
+                }, timeToCall)
+                lastTime = currTime + timeToCall
+                return id
+            }
 
         if (!window.cancelAnimationFrame)
-            window.cancelAnimationFrame = function (id) {
-                clearTimeout(id);
-            };
-    }());
+            window.cancelAnimationFrame = function(id) {
+                clearTimeout(id)
+            }
+    })()
 
     //function to check if image is loaded
     function imageLoaded(img) {
-        return img.complete && (typeof img.naturalWidth === 'undefined' || img.naturalWidth !== 0);
+        return img.complete && (typeof img.naturalWidth === 'undefined' || img.naturalWidth !== 0)
     }
 
-    var imageViewHtml = '<div class="iv-loader"></div> <div class="iv-snap-view">' + '<div class="iv-snap-image-wrap">' + '<div class="iv-snap-handle"></div>' + '</div>' + '<div class="iv-zoom-slider"><div class="iv-zoom-handle"></div></div></div>' + '<div class="iv-image-view" ><div class="iv-image-wrap" ></div></div>';
+    var imageViewHtml =
+        '<div class="iv-loader"></div> <div class="iv-snap-view">' +
+        '<div class="iv-snap-image-wrap">' +
+        '<div class="iv-snap-handle"></div>' +
+        '</div>' +
+        '<div class="iv-zoom-slider"><div class="iv-zoom-handle"></div></div></div>' +
+        '<div class="iv-image-view" ><div class="iv-image-wrap" ></div></div>'
 
     //add a full screen view
-    $(function () {
-        if(!$body.length) $body = $('body');
-        $body.append('<div id="iv-container">' + imageViewHtml + '<div class="iv-close"></div><div>');
-    });
+    document.addEventListener('DOMContentLoaded', function() {
```

Comment:
For assigning event you can also make a helper method, something called 
addEvent, removeEvent, which can get a dom element / dom element list and the event handlers. This will make code more clean